### PR TITLE
Add example of a daemonset forwarding to a deployment

### PR DESCRIPTION
**Pull request by dashpole**
This is a _very_ simple example, which just demonstrates forwarding from one collector to another.  I will follow-up with any additional requested features.

### CODE REVIEW

**Comment by dashpole** [comment]
@jsuereth

File: recipes/daemonset-and-deployment/deployment-collector-config.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      otlp:
+        protocols:
+          grpc:
+          http:
+    processors:
+
+    exporters:
+      googlecloud:
+      logging:
+        loglevel: debug
```
Comment (jsuereth) [inline recipes/daemonset-and-deployment/deployment-collector-config.yaml:32]: Maybe add comments (here and the daemonset) about how you can configure this to lower log-spam.

**Review by jsuereth — APPROVED** [review]
Thanks for the fast turnaround on this example!

File: recipes/daemonset-and-deployment/deployment-collector-config.yaml (hunk 1 above)
Comment (dashpole) [reply recipes/daemonset-and-deployment/deployment-collector-config.yaml]: removed logging

**Pull request merged**